CONTRIBUTING.md

```diff
@@ -83,16 +85,23 @@ There are three places to get help:
 Badge guidelines
 ----------------
 
-- The left-hand side of a badge should not advertize. It should be a noun
-  describing succinctly the meaning of the right-hand-side data.
-- New query parameters must be declared by the service. See
-  `request-handler.js`.
+- The left-hand side of a badge should not advertize. It should be a *noun*
+  succinctly describing the meaning of the right-hand-side data.
```

Comment:
The extra hyphen between "right-hand" and "side" shouldn't be there if I'm not mistaken. ^^

Reply:
I used to look up weird cases like this when I had a subscription to Chicago MOS Online, but no longer…

We could drop the word **data**, and just make it **right-hand side**. That seems just as understandable.